index.html

```diff
@@ -320,41 +340,47 @@ <h3>BrowserCaptureMediaStreamTrack</h3>
                 </p>
                 <ul>
                   <li>
-                    If [=this=] track is [=uncropped=], the user agent MUST only accept [=valid
-                    CropTargets=].
+                    Let |valid| be the result of running [=determine if crop target is valid=] on
+                    |cropTarget|.
+                  </li>
+                  <li>
+                    Let |success| be a [=boolean=], initially set to false.
+                  </li>
+                  <li>
+                    If [=this=] track is [=uncropped=] and |valid| is true, set |success| to true.
```

Comment:
Shall we allow the UA the possibility to fail due to resource limitations?

Reply:
Maybe as a followup?

Reply:
Makes sense to me.